test/CheckoutSdkTest/Payments/Hosted/PaymentsClientTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Checkout.Common;
+using Moq;
+using Shouldly;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Checkout.Payments.Hosted
+{
+    public class PaymentsClientTest : AbstractPaymentsIntegrationTest
```

Comment:
A unit testing class should not extend an integration test class. If the reason to do this was to reuse `CreateHostedPaymentRequest`, don't worry, this is testing code. Please extend `UnitTestFixture` that already contains `ValidDefaultSk`.

Reply:
Ok! I will